docs/the_black_code_style/future_style.md

```diff
@@ -40,3 +40,9 @@ Currently, _Black_ does not split long strings to fit the line length limit. Cur
 there is [an experimental option](labels/experimental-string) to enable splitting
 strings. We plan to enable this option by default once it is fully stable. This is
 tracked in [this issue](https://github.com/psf/black/issues/2188).
+
+## Preview style
+
+Potentially disruptive style changes between major releases are gathered under the
+`--preview` CLI flag. They will eventually become part of main functionality, but until
```

Comment:
"will" seems too strong a word; the intention is that the preview features will have time to incubate and become stable. At the end of each year, we'll have to make a call on whether to promote a feature to stable.

Reply:
Let's do that